src/main/java/frc/robot/subsystems/superstructure/arm/ArmIOSim.java

```diff
@@ -75,7 +75,10 @@ public ArmIOSim(final HardwareConstants.ArmConstants armConstants) {
                 0,
                 Math.PI,
                 true,
-                Units.rotationsToRadians(pivotLowerLimitsRots)
+                ThreadLocalRandom.current().nextDouble(
```

Comment:
What the hell is this??

Reply:
Starts the simulated arm at a random angle instead of always at the lower limit, probably don't need to keep it around but I think its out of scope for this PR.